### ISSUE TITLE
Wildcard support

### ISSUE THREAD

**Issue by r4fx** [vladshcherbin/rollup-plugin-copy#15]
Hi! thank you for nice plugin,
is there any plan to support wildcards to achieve something like below?

```js
 { files: './docs/!*.html', dest: './docs/dist' },
 { files: './docs/robots.txt', dest: './docs/dist' },
 { files: './docs/assets/favicon/!*.*', dest: './docs/dist' },
 { files: './docs/assets/images', dest: './docs/dist/assets/images' },
 { files: './docs/assets/styles', dest: './docs/dist/assets/styles' },
```

**Comment by vladshcherbin** [comment]
Hey,

Yes, I have plans to add glob paths support some time later. Need to test them well too so they don't produce weird results.

**Comment by AnthonySLWhite** [comment]
Here you have it using the glob library that uses 16.1k gzipped:

``` 
import glob from 'glob';
const pluginsList = [
    copy({
      targets: [...glob.sync('public/**/*')],
    })
]
```

Then you just need to add it to your config and export it

**Comment by lazarljubenovic** [comment]
It's probably better to leave this to a third-party lib as @AnthonySLWhite suggests, and just write it as a FAQ in the README, or something with an example or two. Much easier to maintain, much more versatile and configurable, and equally easy to use.

**Comment by vladshcherbin** [comment]
Just a small update about this one:

When writing v1, I actually tried to use [cpy](https://github.com/sindresorhus/cpy) package to copy files which has glob support, but it had some problems I didn't remember. I tried it today and found this problems again (mostly related to creating folders, some mentioned in cpy package issues). 

I'll try to just use plain glob, until then try [solution](https://github.com/vladshcherbin/rollup-plugin-copy/issues/15#issuecomment-485708468) from @AnthonySLWhite. If I won't be able to solve this, I'll add a recipe with this solution to readme as @lazarljubenovic suggested.

Thanks to everyone :)
